packages/api-bindings/src/apollo/__tests__/ApolloCache.spec.ts

```diff
@@ -40,7 +43,7 @@ describe(`Given an instance of the ${ApolloCache.name}`, () => {
         }),
       ],
     });
-    const cache = createApolloCache();
+    const cache = createMockApolloCache();
```

Comment:
Is this intentional? The `createMockApolloCache` exists just to decouple consumer packages from using the real Apollo Cache factory, but this test is about Apollo Cache. I would stick with the real factory function.

Reply:
Fixed